RELEASE/scripts/autoscend/iotms/mr2023.ash

```diff
@@ -106,9 +106,15 @@ void auto_SITCourse()
 {
 	if (!auto_haveSITCourse()) return;
 	//Best choice seems to be insectologist
-	if (!have_skill($skill[insectologist])){
+	if (my_level() < 8 && !have_skill($skill[cryptobotanist])){  //Get cryptobotanist if under level 8
 		use(1,$item[S.I.T. Course Completion Certificate]);
-		//auto_run_choice(1484);
+		//auto_run_choice(1494);
+		return;
+	}
+	if(!get_property("_sitCourseCompleted").to_boolean() && my_level() >= 8) //Get insectologist if over level 8 and haven't SIT'd today
+	{
+		use(1,$item[S.I.T. Course Completion Certificate]);
+		//auto_run_choice(1494);
 		return;
 	}
 }
```

Comment:
These have the same execution block, why not add an `or` case?

Reply:
Good call. Also added a check to avoid additional item use attempts once we get insectologist